src/test_l1_hdmi_cec_driver.c

```diff
@@ -341,7 +354,7 @@ void test_hdmicec_hal_l1_open_logical_address_unavailable_source ( void )
 
      //Here handle = 0 since open failed and close should fail.
     result = HdmiCecClose( handle );
-    if (HDMI_CEC_IO_INVALID_HANDLE  != result) { UT_FAIL_FATAL("close failed"); }
+    CHECK_FOR_EXTENDED_ERROR_CODE(result,HDMI_CEC_IO_INVALID_HANDLE,HDMI_CEC_IO_INVALID_ARGUMENT)
```

Comment:
Let's remove this entire test case as this requires external devices to be connected.  We shall move this test to L3

Reply:
Will consider this in a different PR.